js/app/Omnibox.html

```diff
@@ -165,11 +165,21 @@
   },
   parse: function() {
     let codabar = /^[a-d][0-9]{5,}[a-d]$/, // "a123...789b" pattern
+        json = /^\s*\[\s*{\s*"/,
         parsed,
         value = app.omnibox.element.value;
 
     if (! value) { // nothing was entered, abort
       return;
+    }
+    else if (json.test(value)){
+      // parses the pasted quantity and barcodess
+      let pasted = JSON.parse(value);
+      // confirms non-manual items match the corresponding inventory entry
+      pasted.forEach(utility.getItemFromCache);
```

Comment:
Just an illustration of an idea, obviously cannot work yet without items having methods.
```suggestion
      pasted.forEach(item => {
        if (item.getCached()) {
          app.omnibox.handleItem(item)
        }
      });
```

Reply:
or
```suggestion
      pasted.forEach(utility.item.getFromCache);
```